client/src/pages/ReviewCreate.jsx

```
@@ -145,36 +157,12 @@ export default function ReviewCreate() {
-                  type="file"
-                  accept="image/*"
-                  ref={fileInputRef}
-                  onChange={fileChangeHandler}
-                  style={{ display: 'none' }}
-                />
-              </div>
+              <ImageUploader
+                originalFileName={originalFileName}
+                image={image}
+                setReview={setReview}
+                setUpdatedReview={setUpdatedReview}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)